proto/valve.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+service ValveApi {
```

Comment:
All services, especially traits need docs explaining what they model and what kinds of device they apply to. There's a good chance your trait is not a trait if you can only think of one kind of device it applies to.